core/src/main/java/com/radixdlt/consensus/liveness/PacemakerState.java

```diff
@@ -83,49 +86,53 @@
 public class PacemakerState implements PacemakerReducer {
   private static final Logger log = LogManager.getLogger();
 
-  private final EventDispatcher<RoundUpdate> roundUpdateSender;
   private final ProposerElection proposerElection;
+  private final EventDispatcher<RoundUpdate> roundUpdateSender;
+  private final Metrics metrics;
 
   private Round currentRound;
   private HighQC highQC;
+  private Stopwatch currentRoundStopwatch;
 
   @Inject
   public PacemakerState(
       RoundUpdate roundUpdate,
       ProposerElection proposerElection,
-      EventDispatcher<RoundUpdate> roundUpdateSender) {
+      EventDispatcher<RoundUpdate> roundUpdateSender,
+      Metrics metrics) {
     this.proposerElection = Objects.requireNonNull(proposerElection);
     this.roundUpdateSender = Objects.requireNonNull(roundUpdateSender);
+    this.metrics = metrics;
     this.highQC = roundUpdate.getHighQC();
     this.currentRound = roundUpdate.getCurrentRound();
+    this.currentRoundStopwatch = Stopwatch.createStarted();
```

Comment:
(minor) It feels a bit weird to start a stopwatch during app boot-up (I assume that's when we `@Inject`). AFAIU, the first measured duration may be significantly shorter than the round (if validator booted-up mid round)?

It is probably not a huge measurement skew (i.e. boot-ups are rare), but it may be just more elegant to construct the `PacemakerState` with a stopped stopwatch and start it on first round update?

(if this results in some awkward code, please disregard)

Reply:
yeah, good point, I think it might be better to move this to `Pacemaker` and start the stopwatch in `start`